Web/src/components/games/BalloonShake/index.tsx

```diff
@@ -8,13 +8,15 @@ import {
 import useCounter from '../hooks';
 import { unwrap, createTimerObservable } from '../rxhelpers';
 import { GameState, MotionPermission } from './GameStates';
-import GameCountdown from './GameCountdown';
 import GameDisplay from './GameDisplay';
 import GameResults from './GameResults';
+import BalloonShakeInstructions from './BalloonShakeInstructions';
 import GamePrep from './GamePrep';
+import Countdown from '../Countdown';
 
 const GAME_TIME = 20; // Total shake time given
-const COUNTDOWN_TIME = 4; // Total time to countdown
+const INSTRUCTIONS_TIME = 5; // Total time to read instructions
```

Comment:
Later on we probably should put all these constants in one file

Reply:
I also agree!